src/Form.php

```diff
@@ -1665,21 +1720,29 @@ protected function loadField($element, $group = null, $value = null)
 		{
 			$default = (string) $element['default'];
 
-			if (($translate = $element['translate_default']) && ((string) $translate == 'true' || (string) $translate == '1'))
+			// Try to translate the default value if translations are enabled
+			try
 			{
-				$lang = Language::getInstance();
+				$lang = $this->getText()->getLanguage();
 
-				if ($lang->hasKey($default))
-				{
-					$debug = $lang->setDebug(false);
-					$default = Text::_($default);
-					$lang->setDebug($debug);
-				}
-				else
+				if (($translate = $element['translate_default']) && ((string) $translate == 'true' || (string) $translate == '1'))
 				{
-					$default = Text::_($default);
+					if ($lang->hasKey($default))
+					{
+						$debug = $lang->setDebug(false);
```

Comment:
My only wild guess is if someone was trying to avoid displaying the debug output characters here.  Not really explained well in commit https://github.com/joomla/joomla-cms/commit/70befbea4e61f0de3c847cc5213a5136541585a9 or the tracker item (http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=23711)